CRM/Upgrade/Incremental/php/FourSix.php

```diff
@@ -56,6 +56,9 @@ public function verifyPreDBstate(&$errors) {
    * @param null $currentVer
    */
   public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL) {
+    if (!$this->validateUniqueSettingNames()) {
```

Comment:
@totten Do you think this should be added to the docs as the recommended place for DB pre-checks for minor releases? Docs say that verifyPreDBstate() is not called for minor releases.